acceptance/tests/windows/stop_puppet_svc_stops_puppet.rb

```diff
@@ -0,0 +1,34 @@
+test_name '(C14941) Stopping puppet service on Windows stops puppet'
+
+confine :to, :platform => 'windows'
+
+agents.each do
+  step 'Make sure puppet service is running' do
+    # returns 32 if service is already running
+    result = on(agent, 'sc start puppet', :acceptable_exit_codes => [0, 32])
+    if result.stdout.include?("START_PENDING")
+      sleep 5
+    end
```

Comment:
@joshcooper I made the change your suggested. It worked in 2 of 3 cases, but on the 3rd case I encountered a wrinkle in the way Beaker's `retry_on` method decides when to return. See the FIXME comment in the latest PR. I'm open to suggestions for something better than `sleep`.